app/views/auth/register.phtml

```diff
@@ -3,6 +3,8 @@
 
 	<form method="post" action="<?= _url('user', 'create') ?>">
 		<input type="hidden" name="_csrf" value="<?= FreshRSS_Auth::csrfToken() ?>" />
+		<input type="hidden" name="originController" value="<?= Minz_Request::controllerName(); ?>" />
```

Comment:
```suggestion
		<input type="hidden" name="originController" value="<?= Minz_Request::controllerName() ?>" />
```

I do not think it makes a difference for now, but the short syntax for echo does not take a final semicolon https://php.net/echo

Reply:
I'll fix that